src/test/scala/play/api/cache/redis/configuration/RedisHostSpec.scala

```diff
@@ -15,7 +15,7 @@ class RedisHostSpec extends Specification {
         | host: localhost
         | port: 6379
         |}
-      """
+      """.stripMargin
```

Comment:
This adjustment is redundant and pollutes the code. The tests are designed to be minimal and have the best possible readability and appending `stripMargin` would break it. Thus it's added internally by `WithConfiguration` as you can see.